SalmonKingSeafood/SalmonKingSeafood/public/main.css

```diff
@@ -1,2 +1,48 @@
 ﻿body {
 }
+
+.dropdown-button {
+    padding: 16px;
+    border: none;
+    cursor: pointer;
+    background-color: #f8f8f8;
+    color: #5e5e5e;
+}
+
+.dropdown-content a {
+    color: #5e5e5e;
+    padding: 12px 16px;
+    text-decoration: none;
+    display: block;
+}
+
+@media (min-width: 768px) {
+    .dropdown-content {
+        display: none;
+        position: absolute;
+        min-width: 160px;
+        box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+        z-index: 1;
+    }
+
+    .dropdown:hover .dropdown-content {
+        display: block;
+    }
+}
+
+@media (max-width: 768px) {
```

Comment:
I would strongly advise using the "Mobile First" ideology here. Meaning:
* no media query = smart phones (default)
* min-width then gets wider and wider with subsequent media queries to __overide__ the previous media queries
* there is not going to be many (if any) max-width media queries, as the mobile styling will just be in a smaller min-width media query